index.html

```diff
@@ -286,6 +286,13 @@ <h3 class="my-4">Practical aspects</h3>
                   This combination of offloading and 8-bit optimizers means that we conserve GPU memory (0 bytes per parameter)
                   and also use only a limited amount of CPU memory (2 bytes per parameter).
               </p>
+              <p>
+                  We've created an interactive "calculator" to help you get a better "feel" of which techniques to use and
+                  how they fit together. It covers 8-bit optimizers, offloading, as well as other popular tricks that emphacize memory efficiency.
+                  For each model configuration, it computes GPU and RAM usage based on the shapes of tensors that will be allocated.
+                  Of course, the real-world memory usage depends on the implementation details and technical inefficiencies.
+                  A good rule of thumb for PyTorch is to increase both CPU and GPU memory by 20-30%.
```

Comment:
We have not talked about this, but I think the 20-30% is a good rule of thumb. The models that I tested are in that range of error.